qa/integration/integration_tests.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
 
   # Gem dependencies
   s.add_development_dependency 'elasticsearch'
-  s.add_development_dependency 'childprocess'
+  s.add_development_dependency 'childprocess', '-> 4'
```

Comment:
```suggestion
  s.add_development_dependency 'childprocess', '~> 4'
```

Reply:
I don't think this is a nit, it was a typo on my side :)